walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +270,4 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
-
+}
```

Comment:
?

Reply:
Not completely sure what to fix here. I added the newline back?